src/components/review/ReviewsList.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import axiosInstance from '../../apis/axiosClient';
+import WriteReview from './WriteReview';
+
+const AlbumReviews = ({ albumId }) => {
+  const [reviews, setReviews] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    // Fetch reviews for the given album
+    async function fetchAlbumReviews() {
+      try {
+        const response = await axiosInstance.get(`/reviews/album/${albumId}`);
+        console.log(response.data.allProductReviews);
+
+        const { allProductReviews } = response.data;
+        setReviews(allProductReviews);
+        setLoading(false);
+      } catch (error) {
+        console.error('Error fetching album reviews:', error);
+        setLoading(false);
+      }
```

Comment:
Maybe we want to display some sort of error message to the user (in the div or maybe as an Alert/Snackbar component) if the fetch fails?